### ISSUE TITLE
About the problem of "This file cannot be installed, the file may be damaged" during the installation process

### ISSUE THREAD

**Issue by cocoakeer** [fbelavenuto/arpl#319]
Hello, during the process of installing DS918, when the progress is 88%, the error message "Cannot install this file, the file may be damaged" appears. After the new installation, there is an error message "The hard disk configuration error, please ensure that the hard disk is inserted correctly" error message, It needs to restart and then continue the installation, but it still repeatedly prompts "file corruption" - "hard disk configuration error". I have repeatedly tested this problem.
您好，我在安装DS918过程中进度在88%时出现“无法安装此文件，文件可能已经毁损”错误提示，继续安装后有出现“硬盘配置发生错误，请确保硬盘已正确插入”错误提示，需要重启后再继续安装然后依旧反复提示“文件损毁”-“硬盘配置错误”。我反复测试一直出现此问题。
<img width="600" alt="image" src="https://user-images.githubusercontent.com/112269603/201625300-2df231a1-b7e4-4047-8018-6d11883c4357.png">
<img width="597" alt="image" src="https://user-images.githubusercontent.com/112269603/201625343-5eae6103-3b81-4645-934b-5e9386c21bbf.png">
<img width="873" alt="image" src="https://user-images.githubusercontent.com/112269603/201625368-65b6fc2b-76c5-420d-92c8-1879092bd703.png">

**Comment by cocoakeer** [comment]
I suspect that the version of the startup disk is inconsistent with the version on the official website of Qunhui. I don't know whether this is the problem. Please ask the developer to confirm. Thank you.
我怀疑是制作启动盘的版本和群辉官网版本不一致造成的，不知是否是此问题，麻烦开发者确认下。非常感谢。

**Comment by fbelavenuto** [comment]
What file are you using for installation?
How big are the data disk(s)?

**Comment by cocoakeer** [comment]
> What file are you using for installation? How big are the data disk(s)?

I tested v1.0-beta2 and v0.5-alpha1, and repeated tests showed the same results.
At present, the hard disk size of the installed system is 4TB.
我测试了v1.0-beta2、v0.5-alpha1，反复测试都是同样的结果。
目前安装系统硬盘大小为4TB。

**Comment by fbelavenuto** [comment]
Sorry, what is DSM file used to install, like "DSM_DS918+_42962.pat"?

**Comment by cocoakeer** [comment]
> Sorry, what is DSM file used to install, like "DSM_DS918+_42962.pat"?

Yes, the currently installed version is "DSM_DS918+_42962. pat"?
是的，当前安装的版本为 "DSM_DS918+_42962.pat"?

**Comment by jonathonli** [comment]
Hello. Same problem here (file corruption) while installing DSM under PVE. Tried DSM_DS920+_42962, DSM_DS920+_42218, DSM_DS3622xs+_42962 with matching setting in menu.sh

**Comment by jonathonli** [comment]
> Hello. Same problem here (file corruption) while installing DSM under PVE. Tried DSM_DS920+_42962, DSM_DS920+_42218, DSM_DS3622xs+_42962 with matching setting in menu.sh

My problem solved after change hard drive from scsi to sata. Not sure if it is why.

**Comment by fbelavenuto** [comment]
Ok, then the SCSI driver have some issues, uses the SATA, is more compatible.

**Comment by fbelavenuto** [comment]
Please update ARPL, reboot, update modules, addons and lkm, rebuild the loader and test it.
